### PR TITLE
Tanto tweak and more sheath lunacy

### PR DESCRIPTION
**Pull request by chaosvolt**
The PR changing some knives ( https://github.com/CleverRaven/Cataclysm-DDA/pull/13391 ) has had a few...fun consequences, partly due to poor documentation and warning of this change. The survival knife in particular has caused people to mistake it for a bug both here ( https://github.com/CleverRaven/Cataclysm-DDA/issues/13427 ) and on the forums ( http://smf.cataclysmdda.com/index.php?topic=11291.0 ).
1. Made the Rivtech knife use a scabbard instead of a sheath. SOMEONE forgot it has the same volume as the tanto. Er, the same volume before...
2. Reduced the tanto's volume down to 2 and gave it sheath_knife again.
3. Updated the descriptions of survival knife and Rivtech's knife to mention being unsheathable. Seeing as others have mistaken this change for a bug and complained about it at least twice already, not giving the player some assurance that it's a feature is a Bad Idea.

### CODE REVIEW

File: data/json/items/tools.json [hunk 1]
```diff
@@ -8589,10 +8589,10 @@
     "color": "dark_gray",
     "name": "survival knife",
     "name_plural": "survival knives",
-    "description": "This massive combat knife features a hollow handle with a compass built into the pommel and a row of fearsome looking saw teeth along the back of its blade.",
+    "description": "This massive combat knife features a hollow handle with a compass built into the pommel and a row of fearsome looking saw teeth along the back of its blade.  Being a Bowie knife on steroids, it is too large for most sheaths.",
```
Comment (Coolthulhu) [inline data/json/items/tools.json]: That "combat knife" part could be changed too. It is quite poor when it comes to actually killing stuff.

**Comment by chaosvolt** [comment]
Maybe. We could describe it as a Bowie knife in the first sentence, I'll want to change the "you can't sheath this" message to avoid repetitiveness. XP

**Comment by narc0tiq** [comment]
Or just "This massive knife features...".

**Comment by chaosvolt** [comment]
Either way works, hmm. o.o

**Comment by Rivet-the-Zombie** [comment]
If we're dividing them like this, then we really ought to have options for small sheath, large sheath, and scabbard.

It's ridiculous that you'd need a full-sized scabbard for any knife.

**Comment by chaosvolt** [comment]
That would be nice, yes. That might be beyond the scope of this PR, which is simply cleaning up an inconsistency, fixing tantos via the other available solution, and giving the player more feedback about how the knives work with the current system.

**Comment by kevingranade** [comment]
Calling it a Bowie knife doesn't clarify anything, if you want to emphasize that it's large, just call it large. (which I note it already said, what's the point of calling it a Bowie?)
I'm very against annotating the descriptions with "this item does or does not work in some specific way."  That's a very over-specific thing to put in the description.

**Comment by chaosvolt** [comment]
Hmm. Removing the Bowie part, sure.

NOT including some mention of it not working with sheaths, when it seems logical at first glance to have it fit in a sheath, when it used to fit in a sheath, AND after two separate complaints have been raised in the erroneous belief that it's a bug, doesn't seem as sensible to me.

**Comment by kevingranade** [comment]
Just because someone is complaining doesn't mean they have a valid point.  If it needs clarification, it needs clarification in the sheath, not in every bladed weapon.  Or worse, in an arbitrary subset of the bladed weapons.
Alternately, we can revisit what should and should not fit in a knife sheath.

**Comment by chaosvolt** [comment]
Hmm. Could put the disclaimer in the sheath description I guess. Some form of clarification is needed though, regardless of where it occurs.

**Comment by chaosvolt** [comment]
And thank you for the merge.

**Pull request merged**